pkg/operator/starter.go

```diff
@@ -118,6 +114,19 @@ func RunOperator(ctx context.Context, cc *controllercmd.ControllerContext) error
 		(&resourceapply.ClientHolder{}).WithKubernetes(kubeClient),
 		operatorClient,
 		cc.EventRecorder,
+	).WithConditionalResources(
+		bindata.Asset,
+		[]string{
+			// TODO: remove all of these leader-election entries and files in 4.13
```

Comment:
would need to change the TODO to 4.14 in case we decide not to backport it

Reply:
We switched to leases in 4.11 (https://github.com/openshift/cluster-kube-controller-manager-operator/pull/602). So the create rbac for configmaps is not needed starting that version. Though, I would not temper with it in 4.11 unless there's a high priority request for it. Backporting to 4.12 is fine.

Reply:
Nevertheless, this removes RBAC for the leases for CPC as well. Can you more elaborate (preferable in the PR description) why the RBACs are no longer needed? If possible, linking the code changes which proves the RBACs are no longer needed.

Reply:
regarding the backport, yes I was planning only to 4.12 branch - so it will have a chance to remove the RBAC in 4.12 and we remove it in 4.13

Good idea mentioning the TODOs in the description. I also wanted to link the slack thread but forgot ^^. Updated.